src/utils/package.ts

```diff
@@ -52,11 +55,11 @@ export function formatRepository(repository: any): string | null {
 }
 
 export function formatDate(lastUpdate: string | number): string {
-  return format(new Date(lastUpdate), TIMEFORMAT);
+  return dayjs(new Date(lastUpdate)).format('DD.MM.YYYY, HH:mm:ss');
```

Comment:
You can reuse `TIMEFORMAT` here, it is above 👍

Reply:
Oh True! I forgot 🙂